user/src/main/java/pl/gucio/enzo/chronica/user/data/entity/AccountEntity.java

```diff
@@ -0,0 +1,45 @@
+package pl.gucio.enzo.chronica.user.data.entity;
+
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import pl.gucio.enzo.chronica.user.data.constant.Role;
+
+
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "ACCOUNTS")
+@RequiredArgsConstructor
+@Setter
+@Getter
+public class AccountEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "account_id")
+    private Long id;
+
+    @Column(unique = true, nullable = false)
+    private String username;
+
+    @Column(unique = true, nullable = false)
+    private String mail;
+
+    @Column(unique = true, nullable = false)
+    private Long phoneNumber;
+
+    @Column(nullable = false)
+    private String password;
+
+    private Boolean isActive = true; //TODO dla produkcji ustawic na false
+    private Boolean deprecated;
+    private Role role = Role.USER;
+    private LocalDateTime createdAt = LocalDateTime.now();
```

Comment:
Wydaje mi się że to mogłoby być final skoro i tak ustawiasz to podczas tworzenia

Reply:
//done